ui-lib/src/main/res/ui/restore/values/strings.xml

```diff
@@ -147,23 +149,45 @@
     <string name="ns_security">Security</string>
+    <string name="enter_six_digit_pin_code">Enter Six Digit Pin code</string>
+    <string name="choose_six_digit_pin_code">Choose a Six Digit Pin code</string>
+    <string name="enter_the_same_six_digits">Enter the Same Six Digits</string>
+    <string name="choose_pin_code">Choose Pin code</string>
+    <string name="verify_pin_code">Verify Pin code</string>
+    <string name="wrong_pin_code">Wrong Pin code</string>
+    <string name="set_pin_code_request">Please set the Pin code first</string>
+    <string name="pin_code_setup_done">Pin code set up successfully!</string>
+    <string name="settings_toast_face_touch_id_enabled">Face/Touch ID enabled.</string>
+    <string name="settings_toast_face_touch_id_disabled">Face/Touch ID disabled.</string>
+    <string name="wrong_pin_code_message">Your pin code didn\'t match with the previous pin code.</string>
     <string name="ns_security_text">Set/Change Pin Code &amp; Biometric</string>
+    <string name="set_change_pin_code">Set/Change Nighthawk Pin Code</string>
+    <string name="biometric_backup_phrase_title">Authenticate to proceed</string>
+    <string name="enable_face_id_touch_id">Enable Face ID/Touch ID</string>
+    <string name="dialog_bio_metric_not_enabled_title">Face ID Touch ID Error</string>
+    <string name="dialog_bio_metric_not_enabled_message">Couldn\'t enable this option. It seems that Face ID or Touch ID is not enabled in your mobile device.</string>
     <string name="ns_security_choose">Choose PIN code</string>
     <string name="ns_security_try_again">Please try again</string>
     <string name="ns_about">About</string>
     <string name="ns_about_message">Nighthawk is a Shielded Zcash light-wallet with support for Z-addresses, including sending &amp; receiving shielded transactions, memo support &amp; optional t-addresses support with Auto-Shielding technology.\n\nAs a non-custodial wallet for Zcash, you have sole responsibility over its funds. Please immediately and securely back up the seed words upon creating a wallet.\n\nZcash is a digital currency, or cryptocurrency, like Bitcoin. Zcash was built on the original Bitcoin code base. It was conceived by scientists at MIT, Johns Hopkins and other respected academic and scientific institutions.\n\nNighthawk Wallet requires trust in the default or custom lightwalletd server to display accurate transaction information and CoinGecko service for exchange rate feed. This software is provided "as is", without warranty of any kind, express or implied.</string>
     <string name="ns_view_source">View Source</string>
     <string name="ns_about_text">Nighthawk v%s &amp; Licenses</string>
     <string name="ns_view_license">OSS Licenses</string>
-    <string name="ns_source_code_link">https://github.com/nighthawk-apps/nighthawk-wallet-android</string>
-    <string name="ns_privacy_policy_link">https://nighthawkwallet.com/privacypolicy/</string>
```

Comment:
Keep links within strings.xml instead of static.

Reply:
Strings will be translated in multiple languages so these lines may be duplicate as these are constants

Reply:
Yes, we can use the translations for URLs